tests/bootstrap.php

```diff
@@ -43,7 +43,7 @@ function wpcli_tests_include_config( array $config_filenames = [] ) {
 	if ( $config_filename ) {
 		$config  = file_get_contents( $config_filename );
 		$matches = null;
-		$pattern = '/bootstrap="(?P<bootstrap>.*)"/';
+		$pattern = '/bootstrap="(?P<bootstrap>[^"]*)"/';
```

Comment:
You may still want to replace the `*` with a `+` to not match `bootstrap=""` (which should be rare or non-existent, but saves having to check for it).